app/controllers/identification/IsSimplifiedProcedureController.scala

```diff
@@ -64,7 +65,16 @@ class IsSimplifiedProcedureController @Inject() (
           formWithErrors => Future.successful(BadRequest(view(formWithErrors, mrn, ProcedureType.values, mode))),
           value => {
             implicit val navigator: UserAnswersNavigator = navigatorProvider(mode)
-            IsSimplifiedProcedurePage.writeToUserAnswers(value).writeToSession().navigate()
+            for {
+              firstUa <- Future.fromTry(request.userAnswers.set(IsSimplifiedProcedurePage, value))
+              secondUa <-
+                if (value == ProcedureType.Simplified) {
+                  Future.fromTry(firstUa.set(AuthorisationTypePage(Index(0)), AuthorisationType.ACE))
```

Comment:
Im starting to think we can maybe do this logic on the actual submission of the IE007 rather than here in the controller. As the Authorisation type is always ACE we can also hardcode the dynamic text on the next page. Feel free to grab me to discuss this further.